packages/venia-concept/src/components/ErrorBoundary/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default class ErrorBoundary extends React.Component {
+    state = {
+        hasError: false
+    };
+
+    componentDidCatch() {
+        this.setState({ hasError: true });
+    }
+
+    render() {
+        if (this.state.hasError) {
+            return this.props.fallbackUI;
+        }
+
+        return this.props.children;
+    }
+}
```

Comment:
This is also leftover from when I was providing a fallback UI.

I'm fine including or excluding this file. If we keep it I'll add unit tests to document its usage. Basically:

```jsx
<ErrorBoundary fallbackUI={this.someFunctionThatReturnsJSX}>
  <SomeComponentThatMightThrow />
</ErrorBoundary>
```